### ISSUE TITLE
Citing Skyfield in publications

### ISSUE THREAD

**Issue by bersavosh** [skyfielders/python-skyfield#213]
Thank you for the wonderful package. I was wondering if there is a preferred way to cite the SkyField package?

**Comment by brandon-rhodes** [comment]
Skyfield has never been described in an academic paper so, alas, there is not a source that could be cited as a formal citation! The name "Skyfield" (note the lower case "f"), my own name as the project lead, and the URL of the project here on GitHub would be reasonable identifiers, as would its site URL here:

https://rhodesmill.org/skyfield/

I seem to remember getting this question at least once before, maybe in email. I wonder if there is an existing paper that references Skyfield we could find, and re-use its choice in how it formatted the reference?

Thanks for your interest in attributing the work of the Skyfield project and its contributors!

**Comment by bersavosh** [comment]
Ok, great, thanks! 

Searching ADS, I find at least two papers mentioning the package ([Kounkel et al.](https://ui.adsabs.harvard.edu/#abs/2017ApJ...834..142K/abstract) and [Albert et al.](https://ui.adsabs.harvard.edu/#abs/2018arXiv180711815A/abstract)), both of which seem to simply mention the URL (the latter actually mentions your name as well).

**Comment by brandon-rhodes** [comment]
Then let's consider the URL canonical for the moment. Thanks again for working to cite it properly! :)

**Comment by bersavosh** [comment]
If there isn't a paper in the works on this package, it might be beneficial to obtain a DOI for this project either through [Zenodo](https://zenodo.org/) or [ASCL](https://ascl.net/submissions). This would show the impact of this wonderful package more clearly. :-)

**Comment by brandon-rhodes** [comment]
Good idea! I just did a search for DOI, and it looks like valid URLs are allowed to serve as DOIs, so since two papers have already used it, let's make this the official DOI:

http://rhodesmill.org/skyfield/

Note the "http:", since both papers cited the project before I got around to adding https. Let's keep it "http:" for citations so the match textually. I'll keep this issue open until I can add the DOI to the official documentation. Thanks for the idea!
